app/videobridge/client/actionLink.js

```diff
@@ -7,20 +7,48 @@ import { Rooms } from '../../models';
 import { dispatchToastMessage } from '../../../client/lib/toast';
 
 actionLinks.register('joinJitsiCall', function (message, params, instance) {
-	if (Session.get('openedRoom')) {
-		const rid = Session.get('openedRoom');
+	const rid = Session.get('openedRoom');
+	if (!rid) {
+		return;
+	}
+
+	const room = Rooms.findOne({ _id: rid });
+	const username = Meteor.user()?.username;
+
+	if (!room) {
+		dispatchToastMessage({ type: 'info', message: TAPi18n.__('Call Already Ended', '') });
+		return;
+	}
+
+	if (room?.muted?.includes(username)) {
+		dispatchToastMessage({ type: 'error', message: TAPi18n.__('You_have_been_muted', '') });
+		return;
+	}
+
```

Comment:
```suggestion
	const userId = Meteor.userId();
	const sub = Subscriptions.findOne({ rid, 'u._id': userId });
	if (!sub) {
		Meteor.call('joinRoom', rid);
	}

```
I think we should add the user to the room here, right? If so, we should also add the `Subscriptions` import
Currently, the user is able to join the call, but they don't join the room while doing so.